utils/renderTemplate.js

```diff
@@ -0,0 +1,25 @@
+module.exports = function(msg, template) {
```

Comment:
Could you write a comment here regarding how to use this function?

It seems like `{{ ... }}` is used for expressions in the template. However, it'd be nice to have it written.